src/lib/containers/discussion_forum/DiscussionReply.tsx

```diff
@@ -52,51 +50,48 @@ export const DiscussionReply: React.FC<DiscussionReplyProps> = ({
     currentUserProfile?.ownerId ?? '',
   )
 
-  useEffect(() => {
-    const getReplyMessage = async () => {
-      const messageUrl = await SynapseClient.getReplyMessageUrl(
-        reply.messageKey,
-        accessToken,
-      )
-      const body = await getMessage(messageUrl.messageUrl)
-      setMessage(body)
-    }
-    getReplyMessage()
-  }, [accessToken, reply.messageKey])
-
   return (
     <div className="reply-container">
-      {message && (
-        <div>
-          <UserCard
-            withAvatar={true}
-            avatarSize="MEDIUM"
-            showCardOnHover={true}
-            size={SMALL_USER_CARD}
-            ownerId={reply.createdBy}
-          />
-          <div className="message-body">
-            <MarkdownSynapse markdown={message} objectType={ObjectType.REPLY} />
-            <span>
-              posted {formatDate(moment(reply.createdOn), 'M/D/YYYY')}
-            </span>
-            <div style={{ float: 'right' }}>
-              <button onClick={() => onClickLink()}>
-                <IconSvg icon="link" />
-              </button>
-              {isCurrentUserAuthor && (
-                <button onClick={() => setShowReplyModal(true)}>
-                  <IconSvg icon="edit" />
-                </button>
-              )}
-              {entityBundle?.permissions.canModerate && (
-                <button onClick={() => setShowDeleteModal(true)}>
-                  <IconSvg icon="delete" />
-                </button>
-              )}
+      {isLoading ? (
+        <SkeletonTable numCols={1} numRows={4} />
+      ) : (
+        <>
+          {message && (
+            <div>
+              <UserCard
+                withAvatar={true}
+                avatarSize="MEDIUM"
+                showCardOnHover={true}
+                size={SMALL_USER_CARD}
+                ownerId={reply.createdBy}
+              />
+              <div className="message-body">
+                <MarkdownSynapse
+                  markdown={message}
+                  objectType={ObjectType.REPLY}
+                />
```

Comment:
Probably just need the skeleton here since you have all of the other reply fields

Reply:
When updating, the reply gets rendered. So its original position should be better.